src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -63,6 +63,9 @@ public function getProperties($class, array $context = array())
         foreach ($reflectionClass->getMethods(\ReflectionMethod::IS_PUBLIC) as $reflectionMethod) {
             $propertyName = $this->getPropertyName($reflectionMethod->name);
             if ($propertyName) {
+                if (!isset($properties[$propertyName])) {
+                    $propertyName = lcfirst($propertyName);
```

Comment:
No, I thought that it was:

> Lowercase the first letter unless the string is an uppercased word (eg not only the first letter is uppercased).

For the above proposal, I was refering to getters/setters, for example:

| Input | Result | Wrong |
| ------ | -------- | -------- |
| getSomeVar | someVar | ∅ |
| getSIRET | SIRET | sIRET |
| getIBAN | IBAN | iBAN |
| getAnotherVariableWithUppercase | anotherVariableWithUppercase | ∅ |

That said, I might have misunderstood the goal of your PR.